src/components/Comments.jsx

```diff
@@ -0,0 +1,18 @@
+export function Comments({ comments }) {
+    return (
+      <div>
+  
+      <h4>Comments for article: {comments[0].article_id}</h4>
```

Comment:
This is causing an error for an article with no comment. (For example article 37.)